libc/include/llvm-libc-macros/CMakeLists.txt

```diff
@@ -283,3 +283,9 @@ add_macro_header(
   HDR
     dlfcn-macros.h
 )
+
+add_macro_header(
+  generic_math_macros
```

Comment:
Please change it to `math-function-macros.h` similar to https://github.com/llvm/llvm-project/pull/96008